hubblestack/extmods/returners/splunk_generic_return.py

```diff
@@ -62,6 +98,8 @@ def returner(retdata):
         idx = opts.get('index')
 
         for event in events:
+            event.update({'timezone': __grains__['timezone']})
+            event.update({'timezone_hours_offset': __grains__['timezone_hours_offset']})
```

Comment:
Instead of adding this here (which will require it to be added manually to all of the returners) can you please add it to [stdrec.py](https://github.com/hubblestack/hubble/blob/develop/hubblestack/utils/stdrec.py) (inside of `std_info`) instead? I want to consolidate all of these always-included fields there so we don't have to have so much repeated code.